### DISCUSSION TITLE
Does this allow the use of Vue components within MD for Markdown-it?

### DISCUSSION THREAD

**Discussion by francesco-hayes** [mdit-vue/mdit-vue#3] [Q&A] [answered]
I am trying to find something to render Vue components when calling the markdownIt.render(<string with vue tags>) function. Is this something that accomplishes this because I have had no luck?

Thank you!

**Comment by meteorlxy** [comment, marked as answer]
It won't help. It's only for markdown parsing, and has nothing to do with vue component resolving & rendering.
